### DISCUSSION TITLE
www requests pointing to vercel and email to another server

### DISCUSSION THREAD

**Discussion by piotr-kozlowski-reps** [vercel/community#2458] [Help] [answered]
### Summary

Hi everybody, I'd love to have some help from you with a small issue I can't solve:

Background: 
I've got app deployed on Vercel and an another hosting server with domains/emails etc. etc. I created a subdomain on my hosting server and to connect it with my app on Vercel - I add in DNS  - CNAME - with value as Vercel required 
... but I wish I could still have my emails within that subdomain on my hosting server (not vercel)
Unforunately, my hosting provider told me that a CNAME record redirects also emails to a new server and I can't have emails in that subdomain on his server when that CNAME record is set.

Question:
What is the way to have my subdomain www requests pointing to my app on Vercel but in the same time my emails still taken care within my host provider?

Thank you for help in advance
PEgaz

**Comment by smaeda-ks** [comment, marked as answer]
@piotr-kozlowski-reps You should not set a CNAME on your Apex domain. You can add a A record pointing to Vercel IP address instead:
https://vercel.com/guides/a-record-and-caa-with-vercel

**Reply by piotr-kozlowski-reps** [reply]
Thank you very much. It worked!
